charts/linkerd-control-plane/Chart.yaml

```diff
@@ -16,7 +16,7 @@ dependencies:
 - name: partials
   version: 0.1.0
   repository: file://../partials
-version: 1.7.0-edge
+version: 1.7.1-edge
```

Comment:
should this be a minor bump instead of patch bump because we added `policyController.probeNetworks`?

Reply:
Okay bumped to `1.8.0`.